.github/workflows/ci.yml

```diff
@@ -25,6 +25,12 @@ jobs:
         with:
           fetch-depth: 0
 
+      - uses: actions/cache@v3 
+        id: cache-irfs
+        with:
+          path: ./pyirf/recources/irfs
```

Comment:
I'd not store things in the python package that are not actuall part of the python package...